pkgs/development/ruby-modules/with-packages/test.nix

```diff
@@ -0,0 +1,94 @@
+# a generic test suite for all gems for all ruby versions.
+# use via nix-build.
+let
+  pkgs = import ../../../.. {};
+  lib = pkgs.lib;
+  stdenv = pkgs.stdenv;
+
+  rubyVersions = with pkgs; [
+    # ruby_2_4
+    ruby_2_5
+    # ruby_2_6
+  ];
```

Comment:
Why are the others commented out? Should they either be uncommented or removed?

Reply:
I still run this test manually, so I left it at this for a faster turnaround. 2.4 and 2.5 behave identical, so running 2.4 every time I changed something was a waste of time. But 2.6 has several failures due to the bug in rubygems. Once that is fixed I can uncomment them all, given that I'll only run this once more.

It would be nice to also run those tests via hydra, but I'm unsure how to best expose it.